.github/workflows/ci.yml

```diff
@@ -97,7 +97,8 @@ jobs:
                 format('refs/pull/{0}/head', github.event.client_payload.pull_request.number) || '' }}
       - uses: ./.github/actions/setup
         with:
-          cleanup-node: true
+          cache: 'restore'
```

Comment:
Looks like no job populates the cache anymore, all uses of `./.github/actions/setup` are done with `cache: restore`.